assembl/static2/js/app/graphql/TermsAndLegalNotice.graphql

```diff
@@ -0,0 +1,6 @@
+query TermsAndConditions {
+  legalNoticeAndTerms {
+    legalNotice
```

Comment:
if you don't pass `lang` param, how does this work?

Reply:
You don't need a specific language for this as you just want to know if there is any Terms/LegalNotice available. You think we should still pass lang ?

Reply:
If you download the text just to verify it's not empty to create links, you should really just do the query once, so merge LegalNotice.graphql TermsAndConditions.graphql TermsAndLegalNotice.graphql
There is really too much queries here, what the difference between LegalNoticeAndTerms.graphql and TermsOfUseQuery.graphql ? You should create a fragment and use it in updateLegalNoticeAndTerms.graphql too.

Reply:
What do you mean with the fragment ? update and the query don't act on the same fields, I don't understand

Reply:
@vincentfretin `LegalNoticeAndTerms` query gets the langstring entries (for admin) whereas other queries get the texts in the good lang for display. I think we need `lang` for the check of the footer too because if the admin has not filled the legal notice or terms for the language currently in use, we wont be able to display it (so we don't want the link). I wanted to discuss the necessity of creating `hasLegalNotice` and `hasTerms...` queries to retrieve only a boolean for the footer (as we can assume that most users will not go to these page). What do you think about it ? does it worth it ?

Reply:
If those texts are really long, we can save some kb indeed, so it may be a good idea to do that, and maybe add those hasLegalNotice, hasTerms in TabsCondition query with the others.

Reply:
ok thanks. I thought about adding those to `TabsConditions` but I'm not a big fan since those are not tabs (and will never be). So I will do it in `legalNoticeAndTerms` (or maybe we just have to rename `TabsConditions` ?)

Reply:
I'm in favor of renaming, ActionsQuery?

Reply:
`AvailableLinksQuery` ? 
@paolina92 @ayyazdaniaryan opinions?